docs/k8s-helm.md

```diff
@@ -188,6 +188,17 @@ export RELEASE=timeplus
 helm -n $NS upgrade -f values.yaml $RELEASE timeplus/timeplus-enterprise
 ```
 
+### Update PV size for timeplusd
+
+Due to the [limitation of k8s Statefulset](https://github.com/kubernetes/kubernetes/issues/68737), you will need to manually update the PV size for timeplusd. Notice that this will cause downtime of Timeplus Enterprise.
+
+1. Make sure the `global.pvcDeleteOnStsDelete` is not set or is set to be `false`. You can double check this by running command `kubectl -n <ns> get sts timeplusd -ojsonpath='{.spec.persistentVolumeClaimRetentionPolicy}'` and make sure both `whenDeleted` and `whenScaled` are `retain`. This is extremely important otherwise your PV may be deleted and all the data will be lost.
+1. Run `kubectl -n <ns> delete sts/timeplusd` to temporarily delete the statefulset. Wait until all timeplusd pods are terminated. This step is neccesary to workaround the k8s limitation.
```

Comment:
Just a comment, not have to change.
I usually use 1, 2, 3, not just set 1 for all lines. But I guess markdown renders can handle this case

Reply:
I used to use 1,2,3 and now I think it is better to always use `1`.  Otherwise you will have to update each number if you add/remove an item in the middle.

This shortcut is in standard markdown spec so we don't need to worry about compatibility